src/actions/users.js

```diff
@@ -224,6 +224,8 @@ export function getProfiles(offset, limit = General.PROFILE_CHUNK_SIZE) {
             return null;
         }
 
+        const profiles = profileListToMap(profilesList);
```

Comment:
Oh I think I know you mean. Are you saying pass the action the array then in the store convert to a map so we don't have to convert to a map in a bunch of places? I wouldn't be opposed to that but there is the case where you do have a map in the action and don't want to have to convert map -> array -> map